src/python/deepgnn/graph_engine/snark/dispatcher.py

```diff
@@ -57,6 +59,7 @@ def __init__(
         folder: str,
         parallel: int,
         meta: str,
+        decoder_type: DecoderType,
```

Comment:
why argument reorder is needed?

Reply:
Reverted, only for QueueDispatcher where I do not want to give partition_fn a default value.